incubator/cockroachdb/README.md

```diff
@@ -0,0 +1,184 @@
+was created with your installed chart):
+
+```console
+$ kubectl run -it --rm cockroach-client \
+    --image=cockroachdb/cockroach \
+    --restart=Never \
+    --command -- ./cockroach sql --host my-release-cockroachdb-public
+Waiting for pod default/cockroach-client to be running, status is Pending,
+pod ready: false
+If you don't see a command prompt, try pressing enter.
+root@my-release-cockroachdb-public:26257> SHOW DATABASES;
++--------------------+
+|      Database      |
++--------------------+
+| information_schema |
+| pg_catalog         |
+| system             |
++--------------------+
+(3 rows)
+root@my-release-cockroachdb-public:26257> CREATE DATABASE bank;
+CREATE DATABASE
+root@my-release-cockroachdb-public:26257> CREATE TABLE bank.accounts (id INT
+PRIMARY KEY, balance DECIMAL);
+CREATE TABLE
+root@my-release-cockroachdb-public:26257> INSERT INTO bank.accounts VALUES
+(1234, 10000.50);
+INSERT 1
+root@my-release-cockroachdb-public:26257> SELECT * FROM bank.accounts;
++------+---------+
+|  id  | balance |
++------+---------+
+| 1234 | 10000.5 |
++------+---------+
+(1 row)
+root@my-release-cockroachdb-public:26257> \q
+Waiting for pod default/cockroach-client to terminate, status is Running
+pod "cockroach-client" deleted
+```
+
+## Cluster health
+
+Because our pod spec includes regular health checks of the CockroachDB processes,
+simply running `kubectl get pods` and looking at the `STATUS` column is sufficient
+to determine the health of each instance in the cluster.
+
+If you want more detailed information about the cluster, the best place to look
+is the admin UI.
+
+## Accessing the admin UI
+
+If you want to see information about how the cluster is doing, you can try
+pulling up the CockroachDB admin UI by port-forwarding from your local machine
+to one of the pods (replacing "release-cockroachdb-0" with one of your pods' names):
+
+```shell
+kubectl port-forward release-cockroachdb-0 8080
+```
+
+Once you’ve done that, you should be able to access the admin UI by visiting
+http://localhost:8080/ in your web browser.
+
+## Failover
+
+If any CockroachDB member fails it gets restarted or recreated automatically by
+the Kubernetes infrastructure, and will rejoin the cluster automatically when
+it comes back up. You can test this scenario by killing any of the pets:
+
+```shell
+kubectl delete pod my-release-cockroachdb-1
+```
+
+```shell
+$ kubectl get pods -l "component=my-release-cockroachdb"
+NAME                      READY     STATUS        RESTARTS   AGE
+my-release-cockroachdb-0  1/1       Running       0          5m
+my-release-cockroachdb-2  1/1       Running       0          5m
+```
+
+After a while:
+
+```console
+$ kubectl get pods -l "component=my-release-cockroachdb"
+NAME                      READY     STATUS        RESTARTS   AGE
+my-release-cockroachdb-0  1/1       Running       0          5m
+my-release-cockroachdb-1  1/1       Running       0          20s
```

Comment:
Just to make it look prettier, do you want to make them all running 5m?  Looks like -1 restarted :)